File: 6-alfabeto/alfabeto.c
#include <stdio.h>
#include <ctype.h>

int main() {
    char letra;
    printf("Digite uma letra: ");
    scanf(" %c", &letra);

    letra = tolower(letra);
    if (!isalpha(letra)) {
        printf("Caractere não aceito no programa.\n");
    } else if (letra == 'a' || letra == 'e' || letra == 'i' || letra == 'o' || letra == 'u') {
        printf("A letra é uma vogal.\n");
    } else {
        printf("A letra é uma consoante.\n");
    }

    return 0;
}

File: 6-alfabeto/alfabeto.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var letra string
	fmt.Println("Digite uma letra: ")
	fmt.Scan(&letra)

	letra = strings.ToLower(letra)
	if len(letra) != 1 || (letra[0] < 'a' || letra[0] > 'z') {
		fmt.Println("Caractere não aceito no programa.")
		return
	}

	switch letra {
	case "a", "e", "i", "o", "u":
		fmt.Println("A letra é uma vogal.")
	default:
		fmt.Println("A letra é uma consoante.")
	}
}
